content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md

```diff
@@ -0,0 +1,102 @@
+---
+cover: img.jpg
+title: "Making VSCode run faster on Chromebooks"
+description: "When Linux came to ChromeOS, I was very happy to start using Visual Studio code again as my main text editor. Unfortunately my Chromebook isn’t very fast and Visual Studio code was running very slowly."
+category: Visual Studio Code
+img: https://miro.medium.com/max/1200/1*4qgpMAf4TadK6plhCdxnZg.png
+author: Ada Rose Cannon
+authorImg: https://miro.medium.com/fit/c/96/96/1*Dn8pr_cbYLtc_KfmUNhnBA.png
+tags: [Visual Studio Code, Chrome OS, Development, Web Development, Linux]
+---
+
+# Making VSCode run faster on Chromebooks
+
+When Linux came to ChromeOS, I was very happy to start using Visual Studio code again as my main text editor. Unfortunately my Chromebook isn’t very fast and Visual Studio code was running very slowly.
+
+This was because the visuals were being run in a Chrome instance on Linux then forwarded to ChromeOS which is an expensive thing to do on a HiDPI screen like the Pixelbook.
+
+This seems like a waste of resources to me considering we are already running a Chrome front end in ChromeOS itself. We can take advantage of this by running a version of VSCode which exposes the front end via a http server we can access from ChromeOS.
+
+### Getting started
+
+The project we are using to do this is called [code-server](https://github.com/cdr/code-server):
+[**cdr/code-server**
+*code-server is VS Code running on a remote server, accessible through the browser. Try it out: docker run -it -p…*github.com](https://github.com/cdr/code-server)
+
+Code server is a server which runs the VSCode backend and exposes the front end via http or https. We will run this back-end in crostini then access it through Chrome, so it works entirely offline but through the local browser.
+
+1. Download the latest linux release from [here](https://github.com/cdr/code-server/releases).
+
+1. Extract the code-server binary.
```

Comment:
2. Should be 2 no ?

Reply:
Mark down lists will increment automatically.

`1.` is essentially and <li> in an <ol>